examples/react-native-cli/package.json

```diff
@@ -22,21 +22,17 @@
     "react-native-svg": "^15.3.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.20.0",
-    "@babel/preset-env": "^7.20.0",
-    "@babel/runtime": "^7.20.0",
+    "@babel/core": "^7.25.2",
+    "@babel/preset-env": "^7.25.2",
+    "@babel/runtime": "^7.25.2",
     "@react-native/babel-preset": "0.73.21",
-    "@react-native/eslint-config": "0.73.2",
+    "@react-native/eslint-config": "^0.73.2",
     "@react-native/metro-config": "0.73.5",
-    "@react-native/typescript-config": "0.73.1",
+    "@react-native/typescript-config": "^0.74.0",
```

Comment:
🟡 likely nothing wrong with this version BUT it may not really be the compatibility version - we will need to check this

Reply:
I was getting error, like this

```
error: Error: Unable to resolve module @babel/runtime/helpers/interopRequireDefault from /Users/georgepoladashvili/WebstormProjects/rechunk/src/react-native/index.ts: @babel/runtime/helpers/interopRequireDefault could not be found within the project or in these directories:
```

Reply:
I can revert that

Reply:
yeah let's keep it in 73 range https://github.com/facebook/react-native/blob/9049306394f0b80786857a32982e2c287d951f6e/packages/react-native/template/package.json#L23

Reply:
done